ci/repokitteh/modules/newcontributor.star

```diff
@@ -8,26 +8,52 @@ In the meantime, please take a look at the [contribution guidelines](https://git
 
 """
 
+DRAFT_MESSAGE = """
```

Comment:
i think we should perhaps just include this in the main message

the reason being that until recently i didnt discover how to *create* a draft pr - so habitually created a non-draft and then marked it so

Reply:
hmmm - altho i guess this also sends a message to non-new-contributors - which is kinda confusing given the name of this module

Reply:
yeah, if someone has contributed a few times and then does a draft I think they should get the warning.  I don't think there is a hook for "has ever done a draft before" so it's all or nothing.
moved the module for clarity